src/components/Avatar.js

```diff
@@ -64,12 +64,14 @@ class Avatar extends PureComponent {
             <View pointerEvents="none" style={this.props.containerStyles}>
                 {_.isFunction(this.props.source) || this.state.imageError
                     ? (
-                        <Icon
-                            src={this.state.imageError ? this.props.fallbackIcon : this.props.source}
-                            height={iconSize}
-                            width={iconSize}
-                            fill={this.state.imageError ? themeColors.offline : this.props.fill}
-                        />
+                        <View style={[...imageStyle, {backgroundColor: ''}]}>
```

Comment:
No inline styles.
Links: 

- [Code style and patterns](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md)
- [Styling Components](https://github.com/Expensify/App/blob/main/contributingGuides/STYLING.md)